### PR TITLE
Package beluga.1.0

### PR DESCRIPTION
**Pull request by Ailrun**
### `beluga.1.0`
Implementation of contextual modal logic for reasoning with higher-order abstract syntax

---
* Homepage: https://github.com/Beluga-lang/Beluga
* Source repo: git+https://github.com/Beluga-lang/Beluga.git
* Bug tracker: https://github.com/Beluga-lang/Beluga/issues

---
:camel: Pull-request generated by opam-publish v2.0.2

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
Hi, it seems that you called `opam publish` before the tag was pushed on the repository (at least that's my guess), and thus does not have the required `url` section describing where to get the archive.

Could you re-run `opam publish`? Sorry for the wait.

**Comment by Ailrun** [comment]
Sure, I will try that asap.

**Comment by Ailrun** [comment]
It looks like CI still gives the same error; did I miss something?

**Comment by mseri** [comment]
Indeed, the opam file is stukk missing the `url` stanza. It should have something like
```
url {
  src: "https://..."
  checksum: [
    "md5=..."
    "sha512=..."
  ]
}
```
Usually `opam-publish` and `dune-release` populate it on their own.
Maybe it has some issues with the tag, you should be able to work around it by running
```
opam-publish https://github.com/Beluga-lang/Beluga/archive/v1.0.tar.gz
```

File: packages/beluga/beluga.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "exec" "./TEST"] {with-test}
+  ["dune" "exec" "./TEST" "--" "--" "+htmltest"] {with-test}
+  ["dune" "exec" "./TEST" "--" "--" "+sexp"] {with-test}
+]
+run-test: [
+  ["./TEST"]
+  ["./TEST" "--" "+htmltest"]
+]
```
Comment (kit-ty-kate) [inline packages/beluga/beluga.1.0/opam:33]: The `./TEST` calls seem to repeat themselves twice (in build and in run-test). Is that expected?

**Comment by kit-ty-kate** [comment]
I've fixed your constraints (CI showed that you don't support OCaml < 4.08) and your `dune-project` file shows that you require at least dune 1.1. If I made wrong guesses please correct me. If that's alright, could you return the fix in the opam file upstream?

**Comment by Ailrun** [comment]
Upstream has updated.

**Comment by kit-ty-kate** [comment]
Your `./TEST -- +sexp` test seems to have some issues. Could you have a look at it?

**Comment by kit-ty-kate** [comment]
Thanks a lot!

**Pull request merged**